cluster/images/hyperkube/master-multi.json

```diff
@@ -21,12 +29,19 @@
       "command": [
               "/hyperkube",
               "apiserver",
+              "--admission-control=NamespaceLifecycle,SecurityContextDeny,LimitRanger,ServiceAccount,ResourceQuota",
```

Comment:
Ss of today, we have
```
type SecurityContext struct {
	Capabilities *Capabilities `json:"capabilities,omitempty"`
	Privileged *bool `json:"privileged,omitempty"`
	SELinuxOptions *SELinuxOptions `json:"seLinuxOptions,omitempty"`
	RunAsUser *int64 `json:"runAsUser,omitempty"`
	RunAsNonRoot *bool `json:"runAsNonRoot,omitempty"`
}
```
So it's quite safe to phase out `SecurityContextDeny` IIUC